libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -336,6 +348,15 @@ def _get_invocation_params(
         params["stop"] = stop
         return {**params, **kwargs}
 
+    def _get_ls_params(
+        self,
+        stop: Optional[List[str]] = None,
+        **kwargs: Any,
+    ) -> LangSmithParams:
+        """Get standard params for tracing."""
+        params = self._get_invocation_params(stop=stop, **kwargs)
+        return LangSmithParams(ls_model_type="chat", **params)
```

Comment:
are you expecting this to filter out extra params? it won't

Reply:
I'm expecting it not to filter.